app/helpers/api/v1/shoutouts_helper.rb

```diff
@@ -0,0 +1,2 @@
+module Api::V1::ShoutoutsHelper
```

Comment:
If you're not using a helper please remove it (and its test)

Reply:
Thanks, deleted